de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Average_Product_Prices.sql

```diff
@@ -72,18 +74,24 @@ INNER JOIN C_BPartner bp ON ic.Bill_Bpartner_ID = bp.C_BPartner_ID
 INNER JOIN C_UOM uom ON ic.C_UOM_ID = uom.C_UOM_ID
 INNER JOIN C_UOM price_uom ON ic.Price_UOM_ID = price_uom.C_UOM_ID
 
+LEFT OUTER JOIN C_UOM_Conversion uconv ON uconv.C_UOM_ID = uom.C_UOM_ID
```

Comment:
a `C_UOM_Conversion ` might or might not have an `M_Product_ID`
please either reflect that in the join or use the `uomconvert` DB function

Reply:
used function and show message if the conversion is missing
(unrelated) added movement qty for more visibility on price calc